DDx/DDx/AppDelegate.m

```diff
@@ -153,25 +163,6 @@ - (void)sessionStateChanged
 {
     switch ([PWFramework sharedInstance].client.sessionState)
     {
-        case PWSessionStateFailed:
-        {
-            int errorCode = -1;
-            NSDictionary *userInfo = [[PWFramework sharedInstance].client.acquireException userInfo];
```

Comment:
What about the acquireException information it looks like with your changes we lose that information instead of displaying it to the user.